src/game.cpp

```diff
@@ -7201,9 +7210,8 @@ game::vmenu_ret game::list_items( const std::vector<map_item_stack> &item_list )
         iScrollPos = 0;
 
         if( action == "HELP_KEYBINDINGS" ) {
-            draw_ter();
+            game::draw_ter();
```

Comment:
Minor, but i wouldn't mind keeping this style fix here and just below

Reply:
I didn't realize this was part of your earlier PR. I'll fix that.